pkg/agent/openflow/pipeline_windows.go

```diff
@@ -115,13 +115,13 @@ func (c *client) uplinkSNATFlows(localSubnet net.IPNet, category cookie.Category
 // snatImplementationFlows installs flows that implement SNAT with OVS NAT.
 func (c *client) snatImplementationFlows(nodeIP net.IP, category cookie.Category) []binding.Flow {
 	snatIPRange := &binding.IPRange{StartIP: nodeIP, EndIP: nodeIP}
-	l3FwdTable := c.pipeline[l3ForwardingTable]
-	nextTable := l3FwdTable.GetNext()
+	snatTable := c.pipeline[snatTable]
+	nextTable := snatTable.GetNext()
```

Comment:
@jianjuns @tnqn I am thinking if we should resubmit the packet to the l3DecTTLTable after SNAT on Windows? In current implementation (<= Antrea 1.2), we output the packet to the uplink interface for SNATed ones, and we didn't decrement TTL because we define l2ForwardingCalcTable is the one next to snatTable. I think it is possibly because we output the packet to antrea-gw0 on Linux, and TTL is decremented on the Linux host.

Reply:
That makes sense to me.

Reply:
@wenyingd @jianjuns It looked to me that you are outputting SNATted packet to antrea-gw0 interface (as destination IP is in a different subnet) and not to the uplink interface directly. Once it goes to antrea-gw0, it comes back to OVS pipeline via br-int (as that is the one that has the nodeIP). And then you o/p it via uplink interface (e.g Ethernet). 

You probably meant the same thing. But, I wanted to make sure that I understand the converstion.

Reply:
If you are submitting it directly to uplink interface, how would you know the mac address of this out of cluster IP?

Reply:
@shettyg Thanks for reminding me. You are correct, we actually output the packet th antrea-gw0 even if we perform SNAT inside OVS. The current code for TTL is correct, and please ignore my previous suggestion. I would add comments here for the next table so that we could avoid similar confusion in future.